app/views/provider_interface/provider_relationship_permissions/confirm.html.erb

```diff
@@ -0,0 +1,56 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-xl">Set up permissions</span>
+    <h1 class="govuk-heading-xl">Check permissions</h1>
+
+    <%= form_with(
+      model: @form,
+      url: provider_interface_update_provider_relationship_permissions_path,
+      method: :patch
+    ) do |f| %>
+      <%= f.govuk_error_summary %>
+
+      <h2 class="govuk-heading-m">
+        For courses run by <%= @training_provider_permissions.training_provider.name %> and ratified by <%= @training_provider_permissions.ratifying_provider.name %>
+      </h2>
+
+      <%= f.fields_for :accredited_body_permissions do |abp| %>
+        <%= abp.hidden_field :view_safeguarding_information,
+          value: @form.accredited_body_permissions.view_safeguarding_information %>
+      <% end %>
+
+      <%= f.fields_for :training_provider_permissions do |tpp| %>
+        <%= tpp.hidden_field :view_safeguarding_information,
+          value: @form.training_provider_permissions.view_safeguarding_information %>
+      <% end %>
+
+      <% if @form.training_provider_permissions.view_safeguarding_information %>
+        <p class="govuk-body"><%= @training_provider_permissions.training_provider.name %> can:</p>
+        <ul class="govuk-list">
+          <li>
+            <svg class="app-icon govuk-!-margin-right-1" xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 200 200" focusable="false" fill="currentColor" aria-hidden="true">
+              <path d="M100 200a100 100 0 1 1 0-200 100 100 0 0 1 0 200zm-60-85l40 40 80-80-20-20-60 60-20-20-20 20z"></path>
+            </svg>
+            see safeguarding information
+          </li>
+        </ul>
+      <% end %>
+
+      <% if @form.accredited_body_permissions.view_safeguarding_information %>
```

Comment:
Should we always show nothing if no permissions are granted @adamsilver? Would it be clearer to show show something like "View applications only"?

Reply:
https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/2167/commits/d5321c18db80398f6cf8cb16956ffab879bac21a adds a default, we can iterate this to suit design.